types/env/global.d.ts

```diff
@@ -0,0 +1,3 @@
+export * from '../lh';
+
+export as namespace LH;
```

Comment:
For this to work, we can't expose `LH` globally. This is a temporary measure if we decide to split into multiple PRs.

Reply:
is there an update on this? More PRs to come?

Reply:
The changes in this PR should provide enough typings for the normal exposed api. We should still eliminate this global though.

Doing so would be done in a follow up, and probably wouldn't be a breaking change (although it would touch many files)